### PR TITLE
Call setQuickFixAvailable for SQ

### PR DESCRIPTION
**Pull request by saberduck**
It seems this can't be easily tested, because SonarLint doesn't implement the API

### CODE REVIEW

File: sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/AnalysisProcessor.java [hunk 1]
```diff
@@ -232,7 +232,8 @@ void saveIssue(EslintBridgeServer.Issue issue) {
       newIssue.gap(issue.cost);
     }
 
-    if (isQuickFixCompatible() && issue.quickFixes != null) {
+    if (isQuickFixCompatible() && issue.quickFixes != null && !issue.quickFixes.isEmpty()) {
+      newIssue.setQuickFixAvailable(true);
```
Comment (vilchik-elena) [inline sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/AnalysisProcessor.java:236]: this API is available with SQ only, it's not the same condition as in `isQuickFixCompatible`

**Review by vilchik-elena — CHANGES_REQUESTED** [review]
Build is failing

File: sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/AnalysisProcessor.java (hunk 1 above)
Comment (saberduck) [reply sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/AnalysisProcessor.java:237]: that's just stupid 🤦  fixing it

**Review by vilchik-elena — APPROVED** [review]

**Comment by vilchik-elena** [comment]
@saberduck I think we should add integration test for SQ with quick fix and run ITs on SQ[9.0] to make SC support more robust. WDYT?

**Comment by saberduck** [comment]
> @saberduck I think we should add integration test for SQ with quick fix and run ITs on SQ[9.0] to make SC support more robust. WDYT?

OK, good idea. But maybe we should test LTS and not 9.0?

**Review by vilchik-elena — CHANGES_REQUESTED** [review]
As discussed we need ITs for QF on SQ LTS

**Review by vilchik-elena — APPROVED** [review]

**Pull request merged**